content/home/kubernetes-cluster-step2.md

```diff
@@ -1,8 +1,8 @@
-And deploy Keptn on your cluster
+And deploy Keptn on your Kubernetes cluster
 
 ```console
-keptn install --platform=[aks|eks|gke|openshift|pks|kubernetes]
+keptn install --use-case=[continuous-delivery,quality-gates]
```

Comment:
The `quality-gates` flag is gone. 

Please change to: `keptn install (--use-case=continuous-delivery)`

Reply:
Not so according to the docs: https://keptn.sh/docs/0.7.x/reference/cli/commands/keptn_install/ 
Is it completely gone or just the default?

Reply:
It is the default one. 

Unfortunately, the documentation for the `keptn install` command has not been updated. (FYI: https://github.com/keptn/keptn/blob/master/cli/cmd/install.go#L131)

I will take care of getting the docu for the command fixed.

Reply:
it is now changed